src/TensorFlowNET.Examples.FSharp/BasicModels/LinearRegressionEager.fs

```diff
@@ -0,0 +1,72 @@
+﻿(******************************************************************************
+Copyright 2018 The TensorFlow.NET Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+******************************************************************************)
+
+namespace TensorFlow.NET.Examples.FSharp
+
+open NumSharp
+open Tensorflow
+open type Tensorflow.Binding
+open type Tensorflow.KerasApi
+
+/// <summary>
+/// A linear regression learning algorithm example using TensorFlow library.
+/// https://github.com/aymericdamien/TensorFlow-Examples/blob/master/tensorflow_v2/notebooks/2_BasicModels/linear_regression.ipynb
+/// </summary>
+
+module LinearRegressionEager =
+    let run() =
+        let tf = New<tensorflow>()
+        tf.enable_eager_execution()
```

Comment:
You can ignore `tf.enable_eager_execution` due to since tensorflow 2.0+, it is enabled by default.

Reply:
Got it, thank you.